getting_started/install_openshift.adoc

```diff
@@ -194,7 +194,7 @@ basic authentication, user access, and routes.
 {product-title} provides two command line utilities to interact with it.
 
 * `oc`: for normal project and application management
-* `oadm`: for administrative tasks
+* `oc adm` or `oadm`: for administrative tasks
```

Comment:
@adelton Should we remove the _or `oadm`_ here and leave only `oc adm`? 

Also, should the following line be:
_Use `oc --help` and `oc adm --help` to view all available options.

Reply:
Can we rule out that there might be an environment where `oadm` is available and `oc` is not?

Please note that this pull requests only attempts to actually show that there is `oc adm` way to do things, and prefer that notation. It did not try to convey message that `oadm` is deprecated as it seemed to become in the mean time.

Reply:
@adelton From what I understand, the oadm is being deprecated in 3.9. Should we take out all oadm references now,since you have gotten so many to this point?

Reply:
In 3.9, is it deprecated or completely gone? If it's deprecated but still available (at least in some environments), making note about that with the few `oadm` occurences that are left might give the change better visibility than merely removing them right away.